### ISSUE TITLE
Contrast ratio of some Category Pill do not reach AA accessibility

### ISSUE THREAD

**Issue by RGBboy** [redbadger/pride-london-app#582]
After looking into fixing #155 with #581 I double checked the contrast ratios we are using for these labels. As a result I believe we may have misunderstood reaching AA standard. The table below outlines the standard that the combination of background and foreground color currently produces. Of note are the ones that only reach `AA Large` standard. This means that we should only be using this combination when the font size is 24px or larger for regular text and 18.6px for bold text. Therefore some of the current category labels do not meet the AA standard due to the font size used. Interestingly if we change the foreground color to black we reach the AA standard and can use any font size.

| Label | Background | Foreground | Accessibility Standard | If Foreground Black |
| ----- | ---------- | ---------- | ---------------------- | ------------------- |
| Cabaret and Variety | Coral | White | AA Large | AA |
| Community | Bright Light Blue | Black | AAA | - |
| Talks and Debates | Dark Sky Blue | White | AA Large | AA |
| Film and Screenings | Light Teal | Black | AAA | - |
| Plays and Theatre | Warm Pink | White | AA Large | AA |
| Social and Networking | Turquoise Blue | Black | AA | - |
| Nightlife | Yellow Color | Black | AAA | - |
| Exhibition and Tours | Bright Purple | White | AA | AA |
| Sports and Activities | Vomit Yellow | Black | AAA | - |
| Health | Bubblegum Pink | Black | AAA | - |
| Music | Cornflower Blue | White | AA | AA |

**Comment by RGBboy** [comment]
We have decided not to decrease the font size due to the accessibility implications.
